include/af/vision.h

```diff
@@ -166,6 +166,29 @@ AFAPI void nearestNeighbour(array& idx, array& dist,
  */
 AFAPI array matchTemplate(const array &searchImg, const array &templateImg, const matchType mType=AF_SAD);
 
+
+/**
+   C++ Interface for SUSAN corner detector
+
+   \param[in]  in is input grayscale/intensity image
+   \param[in]  radius Nuclei radius for each pixel neighborhood
```

Comment:
Nuclei is plural, passes the idea that there would be multiple values, I think the singular form nucleus is a better choice.